<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-R8LNRUV" failures="0" tests="7" name="testCases.UpdateTaskContent_SCN04" time="24.133" errors="3" timestamp="2022-10-19T23:36:45 MYT" skipped="0">
  <testcase classname="testCases.UpdateTaskContent_SCN04" name="verfy_header_connection" time="0.020">
    <error message="Cannot invoke &quot;io.restassured.response.Response.header(String)&quot; because &quot;testCases.UpdateTaskContent_SCN04.response&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.header(String)" because "testCases.UpdateTaskContent_SCN04.response" is null
at testCases.UpdateTaskContent_SCN04.verfy_header_connection(UpdateTaskContent_SCN04.java:80)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- verfy_header_connection -->
  <system-out/>
  <testcase classname="testCases.UpdateTaskContent_SCN04" name="verify_status_code" time="0.001">
    <error message="Cannot invoke &quot;io.restassured.response.Response.getStatusCode()&quot; because &quot;testCases.UpdateTaskContent_SCN04.response&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getStatusCode()" because "testCases.UpdateTaskContent_SCN04.response" is null
at testCases.UpdateTaskContent_SCN04.verify_status_code(UpdateTaskContent_SCN04.java:68)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- verify_status_code -->
  <system-out/>
  <testcase classname="testCases.UpdateTaskContent_SCN04" name="verfy_header_content_type" time="0.003">
    <error message="Cannot invoke &quot;io.restassured.response.Response.header(String)&quot; because &quot;testCases.UpdateTaskContent_SCN04.response&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.header(String)" because "testCases.UpdateTaskContent_SCN04.response" is null
at testCases.UpdateTaskContent_SCN04.verfy_header_content_type(UpdateTaskContent_SCN04.java:87)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- verfy_header_content_type -->
  <system-out/>
  <testcase classname="testCases.UpdateTaskContent_SCN04" name="updateTaskContent_verify_response_not_null" time="6.980"/>
  <system-out/>
  <testcase classname="testCases.UpdateTaskContent_SCN04" name="updateTaskContent_verify_response_not_null" time="4.873"/>
  <system-out/>
  <testcase classname="testCases.UpdateTaskContent_SCN04" name="updateTaskContent_verify_response_not_null" time="7.342"/>
  <system-out/>
  <testcase classname="testCases.UpdateTaskContent_SCN04" name="updateTaskContent_verify_response_not_null" time="4.914"/>
  <system-out/>
</testsuite> <!-- testCases.UpdateTaskContent_SCN04 -->
